about.html

```diff
@@ -0,0 +1,168 @@
+  </head>
+
+  <body>
+    <header>
+      <ul
+        class="hidden items-center bg-gray-800 text-white md:flex justify-end h-[30px] pr-[10%]"
+      >
+        <li class="p-1">
+          <a href=""><i class="bi bi-facebook"></i></a>
+        </li>
+        <li class="p-1">
+          <a href=""><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="p-1"><a href="">English </a></li>
+        <li class="p-1"><a href="">My Page</a></li>
+        <li class="p-1"><a href="">logout</a></li>
+      </ul>
+
+      <!-- navigation -->
+
+      <nav class="px-[10%] md:flex h-[55px] justify-between shadow-xl">
+        <i class="menulist bi bi-list md:hidden p-4 text-[40px] pl-0"></i>
+        <a class="p-3 md:flex" href="index.html"
+          ><img src="assets/wine-glass.png" alt="" />wine~up</a
+        >
+        <ul class="items md:flex items-center justify-center">
+          <li class="close"><i class="bi bi-x-lg"></i></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="index.html">Home</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Program</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">Join</a></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Sponsor</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">News</a></li>
+          <li
+            class="item p-1 m-1 hover:text-[#ec5242] border-solid border-4 border-[#ec5242]"
+          >
+            <a href="">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      
+      <!-- about -->
+
+      <section class="bg-[url('assets/desk.png')] bg-contain bg-slate-400">
+        <div
+          class="bg-[#5f5f5f50] flex flex-col text-center items-center justify-items-center py-[5%] gap-10"
+        >
+          <div class="text-[#ec5242]">
+            <h1 class="text-[40px] md:text-[80px] (lato black)">
+              The Grape Escape<br />Wine Tasting and Braai
+            </h1>
+          </div>
+          <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+          <div
+            class="flex bg-white text-[#272a31] px-[10%] py-[5%] text-[#746f6f] mx-[10%] border-solid border-[1px] border-[#d3d3d3]"
+          >
+            <p>
+              Come raise your glass with us on yet another one of our annual
+              parties where we share inside information with future Scientists
+              on how to brew the finest wine in the world,
+            </p>
+          </div>
+          <p class="text-black">
+            Communicate via Email for clarity and reservations!<br />
+            naledi22khosi@gmail.com
+          </p>
+        </div>
+      </section>
+      <hr class="bg-[#ec5242] flex w-[100%] item-center" />
+      <section
+        class="bg-white px-[10%] text-center text-black flex flex-col items-center gap-3"
+      >
```

Comment:
Kindly use the proper overlay color for this section as in the provided design. I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where i compared your design with that of the provided one.